src/page/template/meta.htm

```diff
@@ -1,7 +1,6 @@
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no" />
     <meta name="geoip" country="{{country}}" />
-    <meta property="wire:version" version="{{@config.VERSION}}" />
```

Comment:
Let's please keep this because it's quite handy to use.

Reply:
I don't think this is a good way of checking for the application version. Also it's way easier to check for `window.wire.env.VERSION`.

Reply:
I didn't mean to continue checking from here but to just leave it here for convenience.

Reply:
In general you can find the version also via `/version`, in any resource query below or in `window.wire.env.VERSION` via the console. So there are plenty of places to take the version from.